circle.yml

```diff
@@ -2,12 +2,35 @@ machine:
   services:
     - docker
   environment:
-    TF_IMG: unifio/terraform:0.7.4
-    TF_CMD: docker run -v /home/ubuntu/.aws:/home/user/.aws -e AWS_DEFAULT_REGION=us-east-1 -e LOCAL_USER_ID=1000 --rm
```

Comment:
Typically, yes. However, this causes a lot of warnings in the Circle environment, which is technically docker-in-docker, so I pulled it out.